scheduler/utilization_based_host_allocator.go

```diff
@@ -144,13 +144,19 @@ func evalHostUtilization(ctx context.Context, d distro.Distro, taskGroupData Tas
 	}
 
 	// determine how many free hosts we have that are already up
+	var futureHostFraction float64
+	futureHostFraction = float64(futureHostPercent) / float64(100)
```

Comment:
if at the end of the day we end up calculating a fraction why is it preferable to store as a percentage?

Reply:
Excellent question. A lot of the other similar factors are from 0 to 100, so I decided I would make this a percentage for consistency's sake. I somewhat regret this decision.

Reply:
why do you regret it? 

it seems to me like it adds complexity given we're storing the same value as two different types (float and [int](https://github.com/evergreen-ci/evergreen/pull/4243/files#diff-60c052a6e82755a30b3d3510d99b50a2992eb577e999c51f07eef6971a9d8a5fR218)).
An alternative is to store it as a float in both places. Or we can store it as an int in both places: the migration issue could be addressed by adding a new field to the SchedulerConfig bson (instead of replacing the existing field). Once the change has baked in we can remove the old field from the db. This second alternative would have the advantage that the new bson key will match the field name in the go struct.